Sources/ScipioKit/Producer/PIF/ModuleMapGenerator.swift

```diff
@@ -56,36 +56,45 @@ struct ModuleMapGenerator {
                 return try convertCustomModuleMapForFramework(customModuleMap)
                     .trimmingCharacters(in: .whitespacesAndNewlines)
             case .umbrellaHeader(let headerPath):
-                return """
-                framework module \(context.resolvedTarget.c99name) {
-                    umbrella header "\(headerPath.basename)"
-                    export *
-                }
-                """
-                    .trimmingCharacters(in: .whitespaces)
+                return ([
+                    "framework module \(context.resolvedTarget.c99name) {",
+                    "    umbrella header \"\(headerPath.basename)\"",
+                    "    export *",
+                ]
+                + generateLinkSection(context: context)
+                + ["}"])
+                .joined()
             case .umbrellaDirectory(let directoryPath):
-                return """
-                framework module \(context.resolvedTarget.c99name) {
-                    umbrella "\(directoryPath.basename)"
-                    export *
-                }
-                """
-                    .trimmingCharacters(in: .whitespaces)
+                return ([
+                    "framework module \(context.resolvedTarget.c99name) {",
+                    "    umbrella \"\(directoryPath.basename)\"",
+                    "    export *",
+                ]
+                + generateLinkSection(context: context)
+                + ["}"])
+                .joined()
             case .none:
                 fatalError("Unsupported moduleMapType")
             }
         } else {
             let bridgingHeaderName = "\(context.resolvedTarget.name)-Swift.h"
-            return """
-                framework module \(context.resolvedTarget.c99name) {
-                    header "\(bridgingHeaderName)"
-                    export *
-                }
-            """
-                .trimmingCharacters(in: .whitespaces)
+            return ([
+                "framework module \(context.resolvedTarget.c99name) {",
+                "    header \"\(bridgingHeaderName)\"",
+                "    export *",
+                ]
+                + generateLinkSection(context: context)
+                + ["}"])
+                .joined()
         }
     }
 
+    private func generateLinkSection(context: Context) -> [String] {
+        context.resolvedTarget.dependencies
+            .compactMap(\.target?.c99name)
+            .map { "    link framework \($0)" }
```

Comment:
```suggestion
            .map { "    link framework \"\($0)\"" }
```

It needs to add double quote for framework name


```
NIOTLS.xcframework/ios-arm64/NIOTLS.framework/Modules/module.modulemap:4:20: error: expected framework name as a string
    link framework NIO

```